gtfs/spec/en/reference.md

```diff
@@ -171,6 +171,7 @@ File: **Required**
 |  `shape_id` | ID referencing `shapes.shape_id` | Optional | Identifies a geospatial shape describing the vehicle travel path for a trip. |
 |  `wheelchair_accessible` | Enum | Optional | Indicates wheelchair accessibility. Valid options are:<br><br>`0` or empty - No accessibility information for the trip.<br>`1` - Vehicle being used on this particular trip can accommodate at least one rider in a wheelchair.<br>`2` - No riders in wheelchairs can be accommodated on this trip. |
 |  `bikes_allowed` | Enum | Optional | Indicates whether bikes are allowed. Valid options are:<br><br>`0` or empty - No bike information for the trip.<br>`1` - Vehicle being used on this particular trip can accommodate at least one bicycle.<br>`2` - No bicycles are allowed on this trip. |
+|  run_id | ID | Optional | Identifies the run to which the trip belongs. A run consists of a single trip or many sequential trips driven by the same driver, defined by shared service days and run_id. A run_id can have trips with different service days, making distinct runs. |
```

Comment:
For consistency with the rest of the document, make sure that `run_id` is surrounded by \`s, as in \`run_id\`. This formats the text using a mono-spaced font.

Reply:
Also add "Mid-trip driver changes are indicated in the `stop_times.txt` file."